execution/src/windows/execution.cc

```diff
@@ -313,6 +313,11 @@ namespace leatherman { namespace execution {
                     if (pipe.read) {
                         // Read completed immediately, process the data
                         pipe.buffer.resize(count);
+                        if (convert_newlines) {
+                            pipe.buffer.erase(
+                                std::remove(pipe.buffer.begin(), pipe.buffer.end(), '\r'),
```

Comment:
I contemplated this. I guess it's possible that some program meant for Unix might spit out CRs in places, but have its LFs magically transformed to CRLF pairs. I can defnitely change this to the proper "only convert pairs" logic, it's just a bit more compllicated than a nice algorithm application